libraries/src/Console/AddUserCommand.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\Console;
+
+defined('JPATH_PLATFORM') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\User\User;
+use Joomla\Console\Command\AbstractCommand;
+use Joomla\Filter\InputFilter;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Console command for adding a user
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class AddUserCommand extends AbstractCommand
+{
+	/**
+	 * The default command name
+	 *
+	 * @var    string
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected static $defaultName = 'user:add';
+
+	/**
+	 * SymfonyStyle Object
+	 * @var   object
+	 * @since __DEPLOY_VERSION__
+	 */
+	private $ioStyle;
+
+	/**
+	 * Stores the Input Object
+	 * @var   object
+	 * @since __DEPLOY_VERSION__
+	 */
+	private $cliInput;
+
+	/**
+	 * The username
+	 *
+	 * @var    string
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	private $user;
+
+	/**
+	 * The password
+	 *
+	 * @var    string
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	private $password;
+
+	/**
+	 *  The name
+	 *
+	 * @var    string
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	private $name;
+
+	/**
+	 * The email address
+	 *
+	 * @var    string
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	private $email;
+
+	/**
+	 * The usergroups
+	 *
+	 * @var    array
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	private $userGroups = [];
+
+	/**
+	 * Internal function to execute the command.
+	 *
+	 * @param   InputInterface   $input   The input to inject into the command.
+	 * @param   OutputInterface  $output  The output to inject into the command.
+	 *
+	 * @return  integer  The command exit code
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function doExecute(InputInterface $input, OutputInterface $output): int
+	{
+		$this->configureIO($input, $output);
+		$this->ioStyle->title('Add user');
+		$this->user = $this->getStringFromOption('username', 'Please enter a username');
+		$this->name = $this->getStringFromOption('name', 'Please enter a name (full name of user)');
+		$this->email = $this->getStringFromOption('email', 'Please enter a email address');
```

Comment:
```suggestion
		$this->email = $this->getStringFromOption('email', 'Please enter an email address');
```

You should believe Brian, he is a native English speaker.

Reply:
@richard67 @brianteeman Sorry. I thought this I fixed this already.
I have no intention to ignore Brian.